src/DynamicRupture/FrictionLaws/FrictionSolverCommon.h

```diff
@@ -6,13 +6,41 @@
 #include "Initializer/DynamicRupture.h"
 #include "Kernels/DynamicRupture.h"
 
-namespace seissol::dr::friction_law::common {
 /**
  * Contains common functions required both for CPU and GPU impl.
  * of Dynamic Rupture solvers. The functions placed in
  * this class definition (of the header file) result
  * in the function inlining required for GPU impl.
  */
+namespace seissol::dr::friction_law::common {
+/**
+ * Asserts whether all relevant arrays are properly aligned
+ */
+inline void checkAlignmentPreCompute(
```

Comment:
is it called anywhere?

Reply:
Good point, I forgot to add that :see_no_evil: